package.json

```diff
@@ -36,9 +36,14 @@
     "turndown": "^5.0.1"
   },
   "devDependencies": {
+    "@babel/core": "^7.7.7",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.7.4",
+    "@babel/plugin-proposal-optional-chaining": "^7.7.5",
+    "@babel/preset-env": "^7.7.7",
     "@types/jquery": "^2.0.34",
     "angular": "^1.6.10",
-    "babel-core": "^6.26.0",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
Do we need both `@babel/core` and `babel-core` packages?

Reply:
Thank you, fixed!

Reply:
UPD: we still need `babel-core` for `gulp`